spring-integration-jms/src/main/java/org/springframework/integration/jms/AbstractJmsChannel.java

```diff
@@ -23,9 +23,17 @@
 import org.springframework.util.Assert;
 
 /**
+ * A base {@link AbstractMessageChannel} implementation for JMS-backed message channels.
+ *
+ * <p>When producer and consumer for a JMS destination are in the same application
+ * and process, instead of using a pair of inbound and outbound adapters, it is also
+ * possible to use a single message channel.
```

Comment:
This sentence somehow does not compile in my head.
Plus, it is not true: even if producer and consumer are in different applications, this channel still applies.
Because it is a distributed one.
Another nice aspect of such a channel that several instances of the same application can smoothly use it.

Any chances that we can just remove this sentence?

Another hint: the IDE complains if we use blank lines in the JavaDoc text.
So, if you'd like to make a paragraph, use `<p>` on its own line.
No blank lines at all.